systemtests/scripts/start_minio.sh

```diff
@@ -65,7 +65,7 @@ else
   certs=""
 fi
 
-exec -a "$minio_alias" "${MINIO}" server ${certs} --address ":${minio_port_number}" "$minio_tmp_data_dir" > "$logdir"/minio.log &
+exec -a "$minio_alias" "${MINIO}" server ${certs} --address ":${minio_port_number}" "$minio_tmp_data_dir" > "$logdir"/minio.log 2>/dev/null &
```

Comment:
```suggestion
exec -a "$minio_alias" "${MINIO}" server ${certs} --address ":${minio_port_number}" "$minio_tmp_data_dir" > "$logdir"/minio.log 2>"$logdir"/minio_err.log &
```

Reply:
Its probably better to have that saved somewhere than not!

Reply:
yes perfect